code/modules/mob/mob.dm

```diff
@@ -1105,9 +1107,6 @@
 
 ///Can this mob read (is literate and not blind)
 /mob/proc/can_read(obj/O)
-	if(is_blind())
-		to_chat(src, "<span class='warning'>As you are trying to read [O], you suddenly feel very stupid!</span>")
```

Comment:
naww this is not it. maybe make psyphoza able to read but this is too far lmao

Reply:
🤔

Reply:
hmm maybe